dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -2785,6 +2819,20 @@ Spdp::SpdpTransport::handle_input(ACE_HANDLE h)
         outer_->data_received(data, plist, remote);
         break;
       }
+      case INFO_DST: {
+        if (DCPS::transport_debug.log_messages) {
```

Comment:
See warnings in builds - if this value is `false` I think the correct behavior is indeed what you have written (falling through to `default:`) but some compilers need a specially-formatted comment to avoid a warning when doing this.